client/state/comments/reducer.js

```diff
@@ -126,7 +127,24 @@ export function totalCommentsCount( state = {}, action ) {
 	return state;
 }
 
+/**
+ * Keeps track of if a comment list request is in flight
+ * @param {object} state  - global redux state
+ * @param {object} action - redux action
+ * @returns {boolean} new redux state
+ */
+export function isCommentListLoading( state = false, action ) {
+	switch ( action.type ) {
+		case COMMENTS_LIST_REQUEST:
+			return true;
+		case COMMENTS_RECEIVE: //we may need a more specific success/error action here
+			return false;
+	}
```

Comment:
we're going to be doing lots of comment tracking, possible for different sites than are currently selected.

if we're _going_ to track this here then maybe we can think of a different approach which can give us more reliability.

what if we stored this per-site? that would prevent crossing state for the wrong sites

what if we tracked a count? that would prevent assuming the loading state finishes when we have active requests still

```js
export const siteOpenRequests = ( state = 0, { type } ) => get( [
	[ COMMENTS_LIST_REQUEST ]: state + 1,
	[ COMMENTS_RECEIVE ]: state - 1,
], type, state );

export const openRequests = keyedReducer( 'siteId', openRequests )

const hasOpenRequests = commentState => commentState.openRequests[ siteId ] > 0
```